components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2Authenticator.java

```diff
@@ -172,12 +368,21 @@ public List<Property> getConfigurationProperties() {
 
         List<Property> configProperties = new ArrayList<Property>();
 
+        Property googleOneTap = new Property();
+        googleOneTap.setName(GoogleOAuth2AuthenticationConstant.GOOGLE_ONE_TAP_ENABLED);
+        googleOneTap.setDisplayName("Google One Tap");
+        googleOneTap.setRequired(false);
+        googleOneTap.setType("boolean");
+        googleOneTap.setDescription("Enable Google One Tap as a sign in option.");
+        googleOneTap.setDisplayOrder(1);
```

Comment:
Define something like `int order = 1;`, then do `i++`. It will help for future edits.

Reply:
Since these configurations are hard coded, do we need to add a variable? I feel like it is not good for runtime. +1 if these configurations were not hard coded. What is your opinion one my concern ?

Reply:
My concern is when you need to add some configuration in the future like "Enable One Tap", you have to update multiple places as this number is hard coded. We can't guarantee that we will add it as the last item.

Reply:
Fixed e8155c39133687b8e0f3dce714ead1fd77a8912f